README.adoc

```diff
@@ -359,6 +359,42 @@ There are also two shorthands for adding the attributes `options=""` and `cols="
 With the cursor inside the table, kbd:[+,cols+] adds the latter and kbd:[+,opts+] the former.
 
 
+==== Aligning
+
+When the https://github.com/godlygeek/tabular[Tabular Plugin] is
+installed (which is highly recommended when working with tables),
+alignment of tables is possible via kbd:[+,|+]. This applies only to tables
+that use the Bar kbd:[+|+] as separator between cells.
+
+Be aware that this only works for basic tables and certain features like
```

Comment:
If you really want the user to be aware, use an admonition.
In this case I think it is enough to say, simply:

```
The align feature does not work well for tables with cell spanning.
```

I am not sure what "the cell spanning will not be visualized in the asciidoc sourcecode" means.